src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/items/BaseSTBMachine.java

```diff
@@ -357,7 +357,7 @@ private void buildChargeLabel() {
             }
         }
 
-        s.append(StringUtils.repeat(" ", 15 - s.length()));
+        s.append(Strings.repeat(" ", 16 - s.length()));
```

Comment:
Why was this number increased?

Reply:
Oops! This wasn't supposed to be a part of 1.19 update commits, but basically, when a machine with a sign attached is receiving power, once the power maxes out, STB was throwing an error. It seemed like it was trying to add another "square" to the sign visual. I guessed the math was just 1 off for some reason (sign line length changes? idk). Tested a few machine signs and this stopped throwing the error.